utils/createJestConfig.js

```diff
@@ -14,16 +14,18 @@ module.exports = (resolve, rootDir, isEjecting) => {
     : undefined;
 
   const config = {
-    roots: ['<rootDir>/src'],
+    roots: ['<rootDir>/src', '<rootDir>/test', '<rootDir>/tests'],
 
-    collectCoverageFrom: ['src/**/*.{js,jsx,ts,tsx}', '!src/**/*.d.ts'],
+    collectCoverageFrom: ['src/**/*.{js,jsx,ts,tsx}', 'tests?/**/*.{js,jsx,ts,tsx}','!src/**/*.d.ts'],
 
     setupFiles: [require.resolve('react-app-polyfill/jsdom')],
 
     setupFilesAfterEnv: setupTestsFile ? [setupTestsFile] : [],
     testMatch: [
       '<rootDir>/src/**/__tests__/**/*.{js,jsx,ts,tsx}',
-      '<rootDir>/src/**/*.{spec,test}.{js,jsx,ts,tsx}'
+      '<rootDir>/src/**/*.{spec,test}.{js,jsx,ts,tsx}',
+      '<rootDir>/tests?/**/__tests__/**/*.{js,jsx,ts,tsx}',
+      '<rootDir>/tests?/**/*.{spec,test}.{js,jsx,ts,tsx}',
```

Comment:
I don't think we need all that here:
```
'<rootDir>/tests?/**/__tests__/**/*.{js,jsx,ts,tsx}',
'<rootDir>/tests?/**/*.{spec,test}.{js,jsx,ts,tsx}'
```

`'<rootDir>/tests?/**/*.{js,jsx,ts,tsx}',` is most probably enough as you're already in a test folder

Reply:
done